openmetadata-ui/src/main/resources/ui/src/components/TestSuiteDetails/TestSuiteDetails.component.tsx

```diff
@@ -55,20 +54,23 @@ const TestSuiteDetails = ({
           titleLinks={slashedBreadCrumb}
         />
         <Tooltip
-          title={hasAccess ? t('label.delete') : NO_PERMISSION_FOR_ACTION}>
+          placement="topRight"
+          title={!hasAccess && NO_PERMISSION_FOR_ACTION}>
```

Comment:
same here! please double check we are not showing a tooltip with empty contents

Reply:
https://github.com/open-metadata/OpenMetadata/pull/10034#discussion_r1094208213